packages/cosmic-swingset/types.d.ts

```diff
@@ -0,0 +1,7 @@
+import { SpawnSyncReturns as NodeSpawnSyncReturns } from 'child_process';
+
+declare module 'child_process' {
+  export interface SpawnSyncReturns<T> extends NodeSpawnSyncReturns<T> {
```

Comment:
why was this necessary?

Reply:
We were previously ignoring `packages/cosmic-swingset/check-validator.cjs` when using our old config. Given that it had `check-jessie`, so I assume it was meant to be linted. When trying to lint, I got this error:

```
check-validator.cjs:25:9 - error TS2339: Property 'code' does not exist on type 'SpawnSyncReturns<Buffer<ArrayBufferLike>>'.

25 if (ret.code) {
           ~~~~

check-validator.cjs:26:20 - error TS2339: Property 'code' does not exist on type 'SpawnSyncReturns<Buffer<ArrayBufferLike>>'.

26   process.exit(ret.code);
```

I added this to get the linting working. I'm open to alternatives

Reply:
I'm pretty sure it's a bug. [Node docs for spawnSync](https://nodejs.org/docs/latest-v18.x/api/child_process.html#child_processspawnsynccommand-args-options) and [the TS defs](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/v16/child_process.d.ts#L1417) show a `status: number` property which I think is what this meant to use.

Reply:
Nice catch!